README.md

```diff
@@ -64,6 +64,15 @@ bind-key -n C-l if-shell "$is_vim" "send-keys C-l"  "select-pane -R"
 bind-key -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
 ```
 
+Or optionally, use [TPM][] (tmux plugin manager) installing it first
+and adding the lines below to `~/.tmux.conf`.
```

Comment:
better: "Or optionally, when using the [TPM][] (tmux plugin manager) add the following lines to ..."

Not really sure, but the current version is bumpy ("installing it first").